src/definitions.ts

```diff
@@ -39,13 +38,14 @@ export interface RadarPlugin {
   getContext(options?: Location): Promise<RadarContextCallback>;
   searchPlaces(options: { near?: Location, radius: number, chains?: string[], chainMetadata?: object, categories?: string[], groups?: string[], limit: number }): Promise<RadarSearchPlacesCallback>;
   searchGeofences(options: { near?: Location, radius: number, metadata?: object, tags?: string[], limit: number }): Promise<RadarSearchGeofencesCallback>;
-  autocomplete(options: { query: string, near?: Location, layers?: string[], limit: number, country?: string }): Promise<RadarGeocodeCallback>;
+  autocomplete(options: { query: string, near?: Location, layers?: string[], limit: number, country?: string, expandUnits?: boolean }): Promise<RadarGeocodeCallback>;
+  validateAddress(options: { address: RadarAddress }): Promise<RadarValidateAddressCallback>;
```

Comment:
we need an implementation of this in `web.ts`

Reply:
What's the purpose of `web.ts` -- totally missed it the first time around (it isn't in the notion guide).

Reply:
It's pretty new - basically it's this I think: https://capacitorjs.com/docs/web

We do the same thing on react-native now as well.